core/test/lib/tracehouse/trace-processor-test.js

```diff
@@ -12,7 +12,6 @@ import {readJson} from '../../test-utils.js';
 
 const pwaTrace = readJson('../../fixtures/traces/progressive-app.json', import.meta);
 const badNavStartTrace = readJson('../../fixtures/traces/bad-nav-start-ts.json', import.meta);
-const lateTracingStartedTrace = readJson('../../fixtures/traces/tracingstarted-after-navstart.json', import.meta);
```

Comment:
Turns out `startedAfterNavstartTrace` (which is already required in this test) is the same asset. I liked that name better.